src/curl.cpp

```diff
@@ -188,7 +188,8 @@ const char* BodyData::str(void) const
 //-------------------------------------------------------------------
 // Class S3fsCurl
 //-------------------------------------------------------------------
-#define MULTIPART_SIZE              10485760          // 10MB
+#define MIN_MULTIPART_SIZE          (64*1024)           // 64k
```

Comment:
Minimum multipart size should be 5 MB expect for the last part which can be any size.  Do we need this?